src/DssSpell.t.base.sol

```diff
@@ -1088,10 +1090,13 @@ contract DssSpellTestBase is Config, Test {
         assertEq(clip.wards(address(end)), 1);
 
         // Check toll in/out
-        assertEq(psm.tin(), tin, _concat("Incorrect-tin-", _ilk));
-        assertEq(psm.tout(), tout, _concat("Incorrect-tout-", _ilk));
+        assertEq(psm.tin(), tinBps * WAD / 10000, _concat("Incorrect-tin-", _ilk));
+        assertEq(psm.tout(), toutBps * WAD / 10000, _concat("Incorrect-tout-", _ilk));
 
         uint256 amount = 1000 * (10 ** uint256(token.decimals()));
+        if(amount > line){
+            amount = line;
+        }
```

Comment:
you can avoid the stack too deep and changing all the `tin`, `tout` variables by taking this approach:
```suggestion
        // grab ilk line as amount
        (,,, uint256 amount,) = vat.ilks(_ilk);
        // if line is big, use smaller amount
        if (amount > 1000 * (10 ** uint256(token.decimals()))) {
            amount = 1000 * (10 ** uint256(token.decimals()));
        }
```

Reply:
Thanks this is much better!